File: src/erl/variables.erl
-module(variables).
-export([start/0]).

start() -> 
    put(sentence, "This is the sentence.~n"),

    io:fwrite(get(sentence)),
    io:fwrite(get(sentence)),
    
    put(sentence, "This another sentence."),
    
    io:fwrite(get(sentence)),

    erlang:halt().

File: src/cpp/variables.cpp
#include <iostream>
#include <cstring>

int main()
{
	char sentence[32] = "This is the sentence.";
	std::cout << sentence << std::endl;
	std::cout << sentence << std::endl;
	
	strcpy(sentence, "This is another sentence.");
	std::cout << sentence;
	
	return 0;
}